test/dlc_tests/eks/eks_manifest_templates/kubeflow/install_kubeflow_custom_kfctl.sh

```diff
@@ -1,8 +1,13 @@
 #!/bin/bash
-set -e
+#/ Usage: 
+#/ export AWS_REGION=<AWS-Region>
+#/ ./install_cluster_components.sh eks_cluster_name cluster_autoscalar_image_version
```

Comment:
nit: I think this is the wrong shell script filename

Reply:
> @tejaschumbalkar - this is a lot of infrastructure added to the repo. If someone needs to come make changes to it, it seems we currently don't have automation (functional/unit/integration tests) to ensure that things are working as expected.
> 
> Since this is a somewhat complex system, let's do the following:
> 
> * create a staging cluster for new changes
> * when there are new updates to be deployed, first deploy them to the staging cluster. Then run a series of smoke tests or general infra tests (i.e. ensuring we have the expected k8s versions, ensuring that basic operations (i.e. k get pods) work, etc)
> * When these have succeeded, upgrade the "production" clusters with the appropriate changes
> 
> As we iterate on this, we can also discuss adding unit tests for the code. Most of this is written as shell scripts so it may be a little difficult, but you've done a good job functionalizing most things, so if we want to move to using the k8s python api, it should be fairly straightforward.
> 
> Smoke tests can go in `eks_infra/tests/` for example, since these would be decidedly different from our container tests.
> 
> We should adopt a testing process for adding tests to our `src/` code as well, but that's unrelated and out-of-scope for this project.

Agree. We can add integration checks on the staging cluster that will ensure the functionality of the EKS infrastructure is intact. I will also check on a pythonic option to write these test as this will be applicable only for our infrastructure.